cmd/tk/export.go

```diff
@@ -34,6 +35,9 @@ func exportCmd() *cli.Command {
 	extension := cmd.Flags().String("extension", "yaml", "File extension")
 	merge := cmd.Flags().Bool("merge", false, "Allow merging with existing directory")
 	parallel := cmd.Flags().IntP("parallel", "p", 8, "Number of environments to process in parallel")
+	cachePath := cmd.Flags().StringP("cache-path", "c", "", "URL where cached evaluations should be stored. Supports file:// and gs://")
+	cacheEnvs := cmd.Flags().StringArrayP("cache-envs", "e", nil, "Regexes which define which environment should be cached (if caching is enabled)")
+	warnLongEvals := cmd.Flags().DurationP("warn-long-evals", "w", 0, "If set, will warn when environments take longer than this duration to evaluate")
```

Comment:
This is a bit of an odd CLI argument. Maybe we need a separate benchmark tool in Tanka?

Reply:
Yeah, it is a bit weird. I'll remove it. Maybe we could add tracing support. Something to discuss another time